inc/c_util/constbuffer.h

```diff
@@ -27,13 +29,13 @@ typedef struct CONSTBUFFER_HANDLE_DATA_TAG* CONSTBUFFER_HANDLE;
 typedef struct CONSTBUFFER_TAG
 {
     const unsigned char* buffer;
-    size_t size;
+    uint32_t size;
 } CONSTBUFFER;
 
 typedef void(*CONSTBUFFER_CUSTOM_FREE_FUNC)(void* context);
 
 /*what function should CONSTBUFFER_HANDLE_to_buffer use to allocate the returned serialized form. NULL means malloc from gballoc_hl_malloc_redirect.h of this lib.*/
-typedef void*(*CONSTBUFFER_to_buffer_alloc)(size_t size, void* context);
+typedef void*(*CONSTBUFFER_to_buffer_alloc)(uint32_t size, void* context);
```

Comment:
>uint32_t [](http://example.com/codeflow?start=44&length=8)

wondering if this should still be `size_t` to match malloc... /consider

Reply:
It definitely should be size_t. Until we change our alloc functions to be uint32_t at least, but for now it should be size_t to be aligned with malloc